docs/get_started/plan_deployment/traditional_deployment/user_registry_consideration/plan_realm.md

```diff
@@ -1,19 +1,19 @@
 # Realm support
 
-A realm is a collection of users or groups from one or more branches of your repository tree. Those branches can be part of a single repository, for example an LDAP user registry, or it can be a combination of multiple user registries. A realm is then mapped to a Virtual Portal to allow the realm's user population to log in to the Virtual Portal. This functionality allows you to define areas within HCL Portal that only a limited set of users can access.
+A realm is a collection of users or groups from one or more branches of your repository tree. Those branches can be part of a single repository, for example an LDAP user registry, or it can be a combination of multiple user registries. A realm is then mapped to a virtual portal to allow the realm's user population to log in to the virtual portal. With this functional, you can define areas within HCL Portal that only a limited set of users can access.
```

Comment:
```suggestion
A realm is a collection of users or groups from one or more branches of your repository tree. Those branches can be part of a single repository, for example an LDAP user registry, or it can be a combination of multiple user registries. A realm is then mapped to a virtual portal to allow the realm's user population to log in to the virtual portal. With this functionality, you can define areas within HCL Portal that only a limited set of users can access.
```